website/source/guides/load-balancing/load-balancing.html.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: "guides"
+page_title: "Load Balancing"
+sidebar_current: "guides-load-balancing"
+description: |-
+  There are multiple approaches to load balancing within a Nomad cluster. This
+  discusses the most popular strategies.
+---
+
+# Load Balancing
+
+There are multiple approaches to set up load balancing across a Nomad cluster.
+Most of these methods assume Consul is installed alongside Nomad.
+
+- [Load Balancing Strategies for Consul](https://www.hashicorp.com/blog/load-balancing-strategies-for-consul)
```

Comment:
Can we link to the Consul LB strategies inline and instead use a bullet to link directly to the Fabio guide?

Reply:
How about adding links to Gobetween as well:
http://gobetween.io/

http://gobetween.io/documentation.html#consul